### ISSUE TITLE
[Enhancement]: Click to acknowledge toast notification

### ISSUE THREAD

**Issue by Patriot2407** [advplyr/audiobookshelf#2611]
### Describe the feature/enhancement

So, my work blocks websockets unfortunately. This software is the only web client that still allows me to listen and sync progress with my other devices without websockets working. idk how you managed to achieve this, but kudos from me. Now however, I get an error message every time I use the software that says Socket failed to connect, which covers the playback controls of the interface. I have to go into dev tools every time to remove the element for this toast notification every time, would be nice if there was just an x to acknowledge.

Love the software!

**Comment by advplyr** [comment]
The problem here is web sockets are required for Abs to function as it is intended. If that toast was dismissible then most users would quickly dismiss it instead of fixing the issue with their server. Then we start getting bug reports about progress sync issues for example that would have been resolved had we required the web socket connection.

Can you elaborate on why you are unable to use websockets?

**Comment by Patriot2407** [comment]
I already stated my employer has a firewall that blocks websockets. I have verified this with several other sites that also use websockets that work when I am off my corporate network. If you do not allow this then I will be forced to stop using this software, as this is a huge annoyance do have to edit the notification out using dev tools. Like I said, everything (mostly) works fine without websockets working, I just cannot use the media player controls without hiding that notification
